Changes.md

```diff
@@ -4,16 +4,25 @@
 Improvements
 ------------
 
+- StandardAttributes : Added `shadowedLights` attribute to specify shadow linking.
```

Comment:
I see that we've included it in the node documentation, but do you think it's also worth mentioning here that the intention is for  `ai:visibility:shadow_group` to be replaced by `shadowedLights`?

Reply:
Good idea - squashed into https://github.com/GafferHQ/gaffer/pull/6416/commits/757529c209f3ef06b0516aa5f35a621b1d8f5576.